app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,75 @@
+class TweetsController < ApplicationController
+
+# Show tweets index
+  get '/tweets' do
+    if logged_in?
+      @user = current_user
+      @tweets = Tweet.all
+      erb :'tweets/tweets'
+    else
+      redirect '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :'/tweets/create_tweet'
+    else
+      redirect '/login'
+    end
+  end
+
+  post '/tweet' do
+    if !params[:content].empty?
+      tweet = Tweet.create(content: params[:content])
+      tweet.user_id = current_user.id
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.create(content: params[:content], user: current_user)